src/components/Header/Header.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState } from 'react';
+import { postTodo, USER_ID } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import { Error } from '../../types/Error';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  setTempTodo: (value: Todo) => void,
+  setError: (v: Error) => void,
+  tempTodo: Todo | null,
+};
+
+export const Header: React.FC<Props> = ({
+  setTempTodo,
+  setError,
+  tempTodo,
+}) => {
+  const [title, setTitle] = useState('');
+  const changeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setTitle(value);
+  };
+
+  const createTodo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title) {
+      setError({
+        state: true,
+        type: ErrorType.EmptyTitle,
+      });
+
+      return;
+    }
+
+    setTempTodo({
+      id: 0,
```

Comment:
why do we need to keep id for tempTodo?

Reply:
there is such advice in readme 
![image](https://user-images.githubusercontent.com/90928916/225883962-e469e7c4-32ca-4579-8733-37911b36fd45.png)